### DISCUSSION TITLE
Negotiated Rate Details Object (?)

### DISCUSSION THREAD

**Discussion by Data-san** [CMSgov/price-transparency-guide#185] [Q&A] [answered]
CMS - Under the Negotiated Rate Details Object, can you please change [TIN] from a String to an Array. That would help us reduce prohibitively large. Please advise. Thank you.

**Comment by shaselton-usds** [comment, marked as answer]
Wrapping up some old threads - the `tin` is not an array due to loss of context that would occur when trying to map those values to the `npi` values found in that array.
